src/conductor/op-conductor-ops/launcher.star

```diff
@@ -28,6 +28,46 @@ def launch(
         participants_params=participants_params,
     )
 
+    # List networks
+    _run_op_conductor_ops_command(
+        plan=plan,
+        cmd="list-networks",
+        config_artifact=config_artifact,
+        description="List networks using op-conductor-ops",
+        registry=registry,
+    )
+
+    # Print the network in the config
+    _run_op_conductor_ops_command(
+        plan=plan,
+        cmd="status {}".format(l2_params.network_params.name),
+        config_artifact=config_artifact,
+        description="Status conductors for network {} using op-conductor-ops".format(
+            l2_params.network_params.name
+        ),
+        registry=registry,
+    )
+
+    # Wait a few seconds to see if blocks are progressing or not
+    # This is done for debug purposes
+    plan.run_sh(
+        description="Wait for conductors to stabilize",
+        image=registry.get(_registry.OP_CONDUCTOR_OPS),
+        run="sleep 10",
+        wait=None,
+    )
```

Comment:
please dElEtE if it's just for debugging - if not then `sleep` would definitely produce inconsisten behavior across different machines (and upset developers)